crates/matrix-sdk/src/room/futures.rs

```diff
@@ -298,14 +293,7 @@ impl<'a> IntoFuture for SendAttachment<'a> {
                     let (data, res) = make_thumbnail(data);
 
                     let thumbnail = match res {
-                        Ok((thumbnail_data, thumbnail_info)) => {
-                            data_slot = thumbnail_data;
-                            Some(Thumbnail {
-                                data: data_slot,
-                                content_type: mime::IMAGE_JPEG,
```

Comment:
Nice catch. Is it worth adding a regression test, or too much hassle?

Reply:
I added tests that check this.